docs_src/microservices/device/services/device-gpio/GettingStarted.md

```diff
@@ -0,0 +1,174 @@
+| [device.led.gpio.yaml](https://github.com/edgexfoundry/device-gpio/blob/{{edgexversion}}/cmd/res/profiles/device.led.gpio.yaml)       | [device.led.gpio.yaml](https://github.com/edgexfoundry/device-gpio/blob/{{edgexversion}}/cmd/res/devices/device.led.gpio.yaml)      | Example for GPIO with just LED              |
+
+The device profiles are used to describe the actual GPIO hardware of a device and allow individual GPIOs to be given human-readable names/aliases.
+
+## Example walk-thru
+
+The following are step-by-step examples of using this device service. In these examples, we use Core Command RESTful API to interact with EdgeX rather than directly interact with GPIO device service.
+
+!!! example - "Example - Query Core Command for device `GPIO-Device01` commands"
+    ```shell
+    curl http://localhost:59882/api/v3/device/name/GPIO-Device01
+    ```
+    ```json
+    {
+        "apiVersion": "v2",
+        "statusCode": 200,
+        "deviceCoreCommand": {
+            "deviceName": "GPIO-Device01",
+            "profileName": "Custom-GPIO-Device",
+            "coreCommands": [
+                {
+                    "name": "Power",
+                    "get": true,
+                    "set": true,
+                    "path": "/api/v3/device/name/GPIO-Device01/Power",
+                    "url": "http://edgex-core-command:59882",
+                    "parameters": [
+                        {
+                            "resourceName": "Power",
+                            "valueType": "Bool"
+                        }
+                    ]
+                },
+                {
+                    "name": "LED",
+                    "set": true,
+                    "path": "/api/v3/device/name/GPIO-Device01/LED",
+                    "url": "http://edgex-core-command:59882",
+                    "parameters": [
+                        {
+                            "resourceName": "LED",
+                            "valueType": "Bool"
+                        }
+                    ]
+                },
+                {
+                    "name": "Switch",
+                    "get": true,
+                    "path": "/api/v3/device/name/GPIO-Device01/Switch",
+                    "url": "http://edgex-core-command:59882",
+                    "parameters": [
+                        {
+                            "resourceName": "Switch",
+                            "valueType": "Bool"
+                        }
+                    ]
+                }
+            ]
+        }
+    }
+    ```
+Use the `curl` response to get the command URLs (with device and command ids) to issue commands to the GPIO device via the command service as shown above. You can also use a tool like `Postman` instead of `curl` to issue the same commands.
+
+### Direction setting with sysfs
+When using sysfs, the operations to access and "read" or "write" the GPIO pins are to:
+
+1. Export the pin
```

Comment:
are these zero or 1 indexed?

Reply:
Pin numbers? 
I don't know, but someone setting up GPIO would know the appropriate value.